Segment-Apptimize/Classes/SEGApptimizeIntegration.m

```diff
@@ -1,29 +1,60 @@
 #import "SEGApptimizeIntegration.h"
-
 #import <Apptimize/Apptimize.h>
 #import <Apptimize/Apptimize+Segment.h>
 
-
 @implementation SEGApptimizeIntegration
 
-- (instancetype)initWithSettings:(NSDictionary *)settings analytics:(SEGAnalytics *)analytics apptimize:(id)apptimizeClass;
+static NSString *const APPKEY_SEG_KEY = @"appkey";
+static NSString *const LISTEN_SEG_KEY = @"listen";
+static NSString *const USER_ID_TAG = @"user_id";
+static NSString *const VIEWED_TAG_FORMAT = @"Viewed %@ screen";
+
+- (instancetype)initWithSettings:(NSDictionary *)settings analytics:(SEGAnalytics *)analytics apptimize:(id)apptimizeClass
 {
     if (self = [super init]) {
         _settings = settings;
         _analytics = analytics;
         _apptimizeClass = apptimizeClass;
 
-        [_apptimizeClass startApptimizeWithApplicationKey:[self.settings objectForKey:@"appkey"]];
-        if (![(NSNumber *)[self.settings objectForKey:@"listen"] boolValue]) {
-            [[NSNotificationCenter defaultCenter] addObserver:self
-                                                     selector:@selector(experimentDidGetViewed:)
-                                                         name:ApptimizeTestRunNotification
-                                                       object:nil];
+        NSString *appKey = [self.settings objectForKey:APPKEY_SEG_KEY];
```

Comment:
is this how segment turns us off: by not sending an app key? or they just don't call this method and this check is just for sanity? maybe include a log?